File: content/LLD/Reference-Code/6. Command Pattern/main.py
from chef import Chef
from burger_order import BurgerOrder
from pizza_order import PizzaOrder
from waiter import Waiter

chef = Chef()
burgerOrder = BurgerOrder(chef)
pizzaOrder = PizzaOrder(chef)

waiter = Waiter()
waiter.take_order(burgerOrder)

File: content/LLD/Reference-Code/6. Command Pattern/Main.java
public class Main {
    public static void main(String[] args) {
        Chef chef = new Chef();
        BurgerOrder burgerOrder = new BurgerOrder(chef);
        PizzaOrder pizzaOrder = new PizzaOrder(chef);

        Waiter waiter = new Waiter();
        waiter.takeOrder(burgerOrder);
    }
}
